app/src/main/java/com/github/k409/fitflow/ui/screen/friends/FriendRequestScreen.kt

```diff
@@ -100,7 +112,81 @@ fun FriendRequestContent(
                         null
                     }
                 },
-                label = { Text(stringResource(R.string.send_a_friend_request_to)) },
+                label = { Text(stringResource(R.string.search_by_email)) },
+                modifier = Modifier
+                    .padding(start = 18.dp, end = 12.dp),
+            )
+
+            IconButton(
+                modifier = Modifier
+                    .padding(18.dp),
+                onClick = {
+                    coroutineScope.launch {
+                        foundUser = viewModel.searchUserByEmail(searchText.lowercase())
+                        if (foundUser?.uid == "") {
+                            SnackbarManager.showMessage(context.getString(R.string.user_not_found))
+                        }
```

Comment:
There is an issue where snackbar messages are hidden behind the keyboard. You could try to resolve this by adding `android:windowSoftInputMode="adjustResize"` in the manifest file.